types/object-fit-images/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for object-fit-images 3.2
+// Project: https://github.com/bfred-it/object-fit-images#readme
+// Definitions by: Christopher Parsons <https://github.com/cwparsons>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export interface ObjectFitImagesOptions {
+    watchMQ?: boolean;
+}
+
+export default function objectFitImages(
```

Comment:
The primary entrypoint of `object-fit-images` is https://unpkg.com/object-fit-images@3.2.4/dist/ofi.common-js.js - as such, this should be a

```suggestion
export = objectFitImages;
function objectFitImages(
```

and you may need to set `esModuleInterop` in the tests.